.teamcity/_self/projects/WebApp.kt

```diff
@@ -853,7 +853,7 @@ object QuarantinedE2ETests: BuildType( {
 				export HEADLESS=false
 
 				# Decrypt config
-				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
+				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%E2E_CONFIG_ENCRYPTION_KEY%"
```

Comment:
I renamed these so that previous code in vcs can access the older encryption key without changes. Once everyone is on the new system, one would only need to change the settings.kts file to update keys.